GameServerLib/GameObjects/AttackableUnits/AI/LaneMinion.cs

```diff
@@ -25,6 +25,7 @@ public class LaneMinion : Minion, ILaneMinion
             uint netId = 0
         ) : base(game, null, 0, 0, "", "", 1100, netId)
         {
+            IsLaneMinion = true;
```

Comment:
In a future ticket, we could probably turn this into a base function that returns a static true or false and have LaneMinion override the function to return true. Then we wouldn't need a variable.